src/fs.c

```diff
@@ -904,6 +904,21 @@ void FS_CreatePath(char *path)
 	}
 }
 
+vfsfile_t *FS_OpenFile(char *path)
```

Comment:
I don't like it, use `FS_OpenVFS(path, "rb", FS_GAME)` instead. The only issue with my suggestion is ezquake doesn't have FS_GAME. But whole FS system is a mess so we probably should add support for it in ezquake as well.

Reply:
Didn't notice the existence of that function. I'll drop the commit that introduced the FS_OpenFile and fixup the commit that uses it. Fixes for your other comments will be separate commits for easier review.